addons/account/models/account_move_send.py

```diff
@@ -725,12 +730,18 @@ def _generate_and_send_invoices(self, moves, from_cron=False, allow_raising=True
         if success:
             self._hook_if_success(success)
 
-        # Update sending data of moves
-        for move, move_data in moves_data.items():
-            if from_cron and move_data.get('error'):
-                move.sending_data = {'error': True}
```

Comment:
I probably miss something here because it seems you are removing a feature here

Reply:
This was only used to generate the notification in the caller: https://github.com/odoo/odoo/pull/199092/files#diff-1e3bd6be3bfb83a37ec9fb800ce8b1c95afe0be90ff792874ae7299c320a2f6eL5782
afterwards `sending_data` was set to False anyways.

Now, the `notification` is a function that gets the moves split by the ones in error and the ones that are not in error.

Reply:
I found only usages of `sending_data['error']` in the cron job (and the `if from_cron` makes it quite sure it is used only there). Once the `_generate_and_send_invoices` function  was called, the next step is to `get_account_notification` depending on the error and then set `sending_data = False`.

All of this is still done here without the need to write `sending_data`.